modules/javafx.graphics/src/main/java/javafx/scene/Scene.java

```diff
@@ -3668,6 +3668,7 @@ private void clearPDREventTargets() {
                     ? currentEventTargets.get(0) : null;
             pdrEventTarget.clear();
             pdrEventTargets.clear();
+            fullPDRTmpTargetWrapper.clear();
```

Comment:
the change looks ok to me.
is this the only place where the clear() is needed?

Reply:
Yes, I checked the path of a typical mouse event and this method is always called when a mouse/drag operation is done.

Reply:
ESCAPE key press during dragging?

Reply:
Yes, in this case it will be cleared when the mouse is released.

Reply:
perfect, thank you for verifying.